### DISCUSSION TITLE
Netfilter/iptables xt_u32 module

### DISCUSSION THREAD

**Discussion by maxpain** [fabianishere/udm-kernel-tools#66] [Q&A]
Hello. I need to filter some packets by headers.
```bash
iptables -A FORWARD -p tcp --sport 443 -m u32 --u32 "0x4=0x10000" -m comment --comment "Roskompozor RST" -j DROP
iptables: No chain/target/match by that name.
```
But seems UDMP kernel doesn't have `xt_u32` kernel module. How I can install it?

**Comment by fabianishere** [comment]
You'll need to build this module yourself using [the kernel sources for the UDM/P](https://github.com/fabianishere/udm-kernel). You can either build it for the stock kernel and insert it directly, or build a custom kernel with this module included and launch it with this project.

**Reply by maxpain** [reply]
> You'll need to build this module yourself using [the kernel sources for the UDM/P](https://github.com/fabianishere/udm-kernel). You can either build it for the stock kernel and insert it directly, or build a custom kernel with this module included and launch it with this project.

Could you please make some example how to do that?

**Reply by fabianishere** [reply]
Sorry, I won't have time to provide more details than this. There are resources available online on how to do this.
